anng/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "anng"
+version = "0.1.0"
+edition = "2024"
+license = "MIT OR Apache-2.0"
```

Comment:
I'm not up to date. Do we need explicit `LICENSE` files here? [mit](https://opensource.org/license/mit) and [apache-2](https://opensource.org/license/apache-2-0)